libraries/Microsoft.Bot.Builder/PathUtils.cs

```diff
@@ -9,22 +9,22 @@ public static class PathUtils
         /// </summary>
         /// <remarks>
         /// path is from authored content which doesn't know what OS it is running on.
-        /// This method treats / and \ both as seperators regardless of OS, for windows that means / -> \ and for linux/mac \ -> /.
-        /// This allows author to use ../foo.lg or ..\foo.lg as equivelents for importing.
+        /// This method treats / and \ both as separators regardless of OS, for windows that means / -> \ and for linux/mac \ -> /.
+        /// This allows author to use ../foo.lg or ..\foo.lg as equivalents for importing.
         /// </remarks>
-        /// <param name="ambigiousPath">authoredPath.</param>
+        /// <param name="ambiguousPath">authoredPath.</param>
         /// <returns>path expressed as OS path.</returns>
-        public static string NormalizePath(string ambigiousPath)
+        public static string NormalizePath(string ambiguousPath)
```

Comment:
I'm afraid we can't take this change, as it would be a technically breaking change. Anyone calling this as a named parameter would be broken. 

The other spelling fixes would be great though!

Reply:
Ok, I see, but what is a sense to name param which is only one in the method?

Reply:
Changed to 'ambigiousPath' as it was